lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -201,7 +206,7 @@ serveWallet
 serveWallet
     Tracers{..} sTolerance databaseDir hostPref listen lj beforeMainLoop = do
     installSignalHandlers (traceWith applicationTracer MsgSigTerm)
-    traceWith applicationTracer MsgStarting
+    traceWith applicationTracer $ MsgStarting lj
```

Comment:
`lj` is here the only one name not self-descriptive (ie., in 207 line)

Reply:
OK fixed - I was trying to stay under 80 columns. :grin: